server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -661,7 +663,11 @@ func (e *EventChannel) handleEvent(event *pepb.PublishBuildToolEventStreamReques
 				return err
 			}
 			if e.env.GetConfigurator().GetStorageEnableChunkedEventLogs() {
-				e.logWriter = eventlog.NewEventLogWriter(e.ctx, e.env.GetBlobstore(), e.env.GetKeyValStore(), iid)
+				linesToRetain, err := getCursesLinesFromStartedBuildEvent(&bazelBuildEvent)
```

Comment:
It feels a little scary to me to depend on this certain flag in the build started event, otherwise the whole bes will be an error. Could we have a sane default here (whatever it was before this CL) and then optionally attempt to use the value from the event?

Reply:
Yup, no problem; done.